private_sharing/models.py

```diff
@@ -408,6 +409,8 @@ class Meta:  # noqa: D101
         verbose_name="Deauthorization Webhook URL",
     )
 
+    diy_project = models.BooleanField(default=False)
```

Comment:
Minor, but I'd prefer diyexperiment to be consistent with the app we've started making. :) (OK if we change the name later, but we should be consistent about using the current working title.)